docs/src/libraries/evm/ibc_eureka_transfer.md

```diff
@@ -0,0 +1,107 @@
+# IBC Eureka Transfer Library
+
+The IBC Eureka Transfer library enables transferring ERC20 tokens from an **input account** on an EVM chain to a **recipient** on an IBC Eureka connected chain using the IBC protocol via [IBC Eureka's Solidity implementation](https://docs.skip.build/go/eureka/eureka-overview). This library also takes advantage of [Skip Go API](https://docs.skip.build/go/general/getting-started) that provides an `EurekaHandler` wrapper and the relaying service for the underlying protocol. It is typically used as part of a **Valence Program**. In that context, a **Processor** contract will be the main contract interacting with the IBC Eureka Transfer library.
```

Comment:
In CosmWasm we have a LibraryAccount type to deal with various addresses. We don't have a similar concept in Ethereum. I'm wondering how we are going to link the recipient address to a Valence Program account.

This is not a problem specific to this library but something we need to think about more generally. @vlopes11 will have to consider this for the vault deployer